code_samples/paging_predicate_sample/README.md

```diff
@@ -0,0 +1,10 @@
+# hazelcast-paging-predicate-example
+
+1. Download hazelcast. Extract it somewhere.  https://hazelcast.org/imdg/download/
```

Comment:
This needs a general fix.

First, we need to change the wording here so it doesn't contain `I` etc. and make it a bit more formal.

I also would like to see what is this code sample for, and why do we need to do such things etc.